File: Composition/CS/Composition/Algorithm/Composition/LowPassFilter.cs
using System.Collections.Generic;
using System.Linq;

namespace Algorithm.Composition
{
    public class LowPassFilter : IMeasurementFilter
    {
        public IEnumerable<Measurement> Filter(IEnumerable<Measurement> measurements)
        {
            return measurements.Where(m => m.X < 100 & m.Y < 100);
        }
    }
}

File: Composition/Java/Composition/src/main/java/algorithm/composition/LowPassFilter.java
package algorithm.composition;

import java.util.Collection;

import algorithm.Measurement;
import algorithm.MeasurementUtil;

public class LowPassFilter implements IMeasurementFilter {

	@Override
	public Collection<Measurement> filter(Collection<Measurement> measurements) {
		return MeasurementUtil.WhereLessThanXandY(measurements, 100, 100);
	}

}
